tests/no_wrapper/multiprocess_tests.py

```
@@ -30,6 +30,19 @@ def _check_process(port):
     assert int(port) == food._connection_kwargs["port"]
 
 
+class GetDbConnTests(unittest.TestCase):
+    def test_get_db_conn(self):
+        import fiftyone.core.odm as foo
+
+        conn = foo.get_db_conn()
+        self.assertIsNotNone(conn)
+        conn.client.close()
+        assert conn.client._closed == True
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `conn.client._closed:` for truth checks

Replace with `conn.client._closed`

(E712)